app/src/main/java/com/example/keniphhomemaintenance/maintenance_screen/MaintenanceView.kt

```diff
@@ -0,0 +1,252 @@
+            painter = painterResource(R.drawable.ic_baseline_build_24), 
+            contentDescription = "Maintenance Item Type",
+            modifier = Modifier
+                .size(dimensionResource(R.dimen.maintenance_image_size))
+                .clip(CircleShape)
+        )
+        
+        Spacer(modifier = Modifier.width(dimensionResource(R.dimen.maintenance_image_end_padding)))
+
+        Column {
+            Text(
+                text = maintenanceItem.title,
+                color = MaterialTheme.colors.primaryVariant,
+                style = MaterialTheme.typography.h5
+            )
+            Spacer(modifier = Modifier.height(dimensionResource(R.dimen.maintenance_title_and_due_date_padding)))
+            Text(
+                text = maintenanceItem.dueDate,
+                color = Color.Black,
+                style = MaterialTheme.typography.body2
+            )
+            Text(
+                text = maintenanceItem.location,
+                color = Color.Black,
+                style = MaterialTheme.typography.body2
+
+            )
+        }
+    }
+}
+
+@Composable
+fun MaintenanceItemsList(maintenanceItems: List<MaintenanceItem>) {
+    LazyColumn(
+        modifier = Modifier
+            .fillMaxHeight()
+            .padding(bottom = dimensionResource(R.dimen.maintenance_item_list_bottom_padding))
+    ) {
+        items(maintenanceItems) { maintenanceItems ->
+            MaintenanceCard(maintenanceItems)
+        }
+    }
+}
+
+@Composable
+fun ShowDialog(isDialogOpen: MutableState<Boolean>) {
+    val itemTitle = remember { mutableStateOf("") }
+    val dueDate = remember { mutableStateOf("") }
+    val location = remember { mutableStateOf("") }
+
+    if (isDialogOpen.value) {
+        Dialog(
+            onDismissRequest = { isDialogOpen.value = false }) {
+            Surface(
+                modifier = Modifier
+                    .fillMaxSize()
+                    .padding(vertical = dimensionResource(R.dimen.maintenance_item_dialog_vertical_padding)),
+              shape = RoundedCornerShape(dimensionResource(R.dimen.maintenance_item_dialog_corner_radius)),
+                color = Color.DarkGray
+            ) {
+                val focusManager = LocalFocusManager.current
+                Column(
+                    modifier = Modifier.padding(dimensionResource(R.dimen.maintenance_item_dialog_top_padding)),
+                    horizontalAlignment = Alignment.CenterHorizontally,
+                    verticalArrangement = Arrangement.Top
+                ) {
+                    Text(
+                        text = stringResource(R.string.maintenance_item_dialog_title),
+                        color = Color.White,
+                        fontWeight = FontWeight.Bold,
+                        fontSize = 25.sp
+                    )
+                    Spacer(modifier = Modifier.padding(dimensionResource(R.dimen.maintenance_item_dialog_title_bottom_padding)))
+                    
```

Comment:
Have not implemented anything yet which requires the user to enter a value, or close the dialog. Will handle in the future. Applies to the two OutlinedTextFields below as well

Reply:
But what if I wanted to fix my   by   at  ?